mne/channels/channels.py

```diff
@@ -560,14 +560,18 @@ def inst_has(attr):
         elif isinstance(self, Evoked):
             self.data = self.data.take(idx, axis=0)
 
-    def add_channels(self, add_list, copy=None):
+    def add_channels(self, add_list, force_update_info=False, copy=None):
         """Append new channels to the instance.
 
         Parameters
         ----------
         add_list : list
             A list of objects to append to self. Must contain all the same
             type as the current object
+        force_update_info : bool
+            If True, force the info for objects to be appended to match the
+            values in `self`. This should generally only be used when adding
+            stim channels for which important metadata won't be overwritten.
```

Comment:
add `..versionadded:: 0.12` with newlines on either side to render properly (see existing examples)